bin/configurer/utils/idam-create-service.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -eu
+
+dir=$(dirname ${0})
+
+LABEL="${1:-ccd_gateway}"
```

Comment:
Script is reused for two services now. IMO it should not have defaults for arguments that matter and require change. I'd rather fail fast

Reply:
Done.